db/discord_db/channel.py

```diff
@@ -0,0 +1,13 @@
+from base import Base
+
+from sqlalchemy import String
```

Comment:
Move all modules to a module.py file instead of a file for each module.
File for each class relevant only for Python functionality.